README.md

```diff
@@ -626,3 +626,7 @@ access them from the IR
 * Tables with multiple apply calls
 
 See also [unsupported P4_16 language features](backends/bmv2/README.md#unsupported-p4_16-language-features).
+
+# ⭐ How to contribute
```

Comment:
The star looks strange in the README, it is very large and only used here. I would remove it.

Reply:
Sure !